include/swift/AST/SearchPathOptions.h

```diff
@@ -476,7 +476,7 @@ class SearchPathOptions {
   std::vector<std::string> CandidateCompiledModules;
 
   /// A map of explicit Swift module information.
-  std::string ExplicitSwiftModuleMap;
+  std::string ExplicitSwiftModuleMapPath;
```

Comment:
The caching build doesn't really take a path here (but a CASID). Maybe it wasn't a good idea to reuse this flag for caching build for the first place.